api/container/containerv1/clusters.go

```diff
@@ -31,6 +31,20 @@ type ClusterInfo struct {
 	State             string
 	IsPaid            bool
 	WorkerCount       int
+	Vlans             []Vlan
+}
+
+type Vlan struct {
+	ID      string
+	Subnets []SubnetV
+}
+
+type SubnetV struct {
```

Comment:
Can we call it just Subnet?

Reply:
Subnet is already used . So added V denoting Vlan